tests/IntegrationTest.php

```diff
@@ -66,6 +65,8 @@ public function testOrderLifecycle()
 
         $checkoutSession = $checkoutSessionResponse->asJson();
 
+        print_r($checkoutSession);
```

Comment:
Q: Should we remove this line?

Reply:
This is intended to show the session data :) Sometimes we want the id and use other tools to verify it.